lib/parsers/get-yarn-lock-dep-graph.ts

```diff
@@ -0,0 +1,150 @@
+import { Dep, getTopLevelDeps, LockfileType, ManifestFile } from './index';
+import { DepGraphBuilder } from '@snyk/dep-graph';
+import * as pMap from 'p-map';
+import { YarnLock } from './yarn-lock-parse';
+import { eventLoopSpinner } from 'event-loop-spinner';
+import { OutOfSyncError } from '../errors';
+import {
+  EVENT_PROCESSING_CONCURRENCY,
+  YarnLockDep,
+} from './yarn-lock-parse-base';
+
+type VersionHolder = YarnLockDep | { version: string };
```

Comment:
let's move types to a unified place

Reply:
It's only for internal use in this file
There's no reason to create a centric types file for types that are not being used outside of a specific file